CHANGELOG.md

```diff
@@ -4,6 +4,59 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0-rc1 (2020-10-06)
+
+This release adds major new features and bugfixes since the 1.7.4 release.
+We deem it moderate priority for upgrading.
+
+This release adds the long-awaited support for distributed hypertables to
+TimescaleDB. This release also adds support for user-defined actions allowing
+users to define actions that are run by the TimescaleDB automation framework.
+
+In addition to these major new features, the 2.0 branch introduces _breaking_ changes
+to APIs and existing features, such as continuous aggregates. These changes are not
+backwards compatible and might require changes to clients and/or scripts that rely on
+the previous APIs. Please review our updated documentation and do proper testing to
+ensure compatibility with your existing applications.
+
+The noticeable breaking changes in APIs are:
+- Function rename and arguments redesign for compression and retention policies
```

Comment:
I'm not sure exactly how to parse `Function rename and arguments redesign`, maybe something simpler like:

```suggestion
- Redefined functions for compression and retention policies
```